graphql_api/types/repository/repository.py

```diff
@@ -579,8 +584,22 @@ async def resolve_test_results(
     filters=None,
     **kwargs,
 ):
-    queryset = await sync_to_async(aggregate_test_results)(
-        repoid=repository.repoid, branch=filters.get("branch") if filters else None
+    parameter = None
```

Comment:
do we need to handle a default case here?

Reply:
the default case is a noop so i don't think so